.jazzy.yaml

```diff
@@ -0,0 +1,14 @@
+module: HTMLEntities
+author: IBM
+github_url: https://github.com/IBM-Swift/swift-html-entities
+
+theme: fullwidth
+clean: true
+exclude: [Packages]
+
+readme: README.md
+
+skip_undocumented: false
+hide_documentation_coverage: false
+
+xcodebuild_arguments: [-project, HTMLEntities.xcodeproj, -target, HTMLEntities, LIBRARY_SEARCH_PATHS=.build/debug]
```

Comment:
We don't need `LIBRARY_SEARCH_PATHS=.build/debug` any more. This should just be:

`xcodebuild_arguments: [-project, HTMLEntities.xcodeproj, -target, HTMLEntities]`

I think.